tests/versions/python.yml

```diff
@@ -4,5 +4,4 @@ PYTHON_AGENT:
   # release;release -> pip install elastic-apm==version
   - 'github;master'
   - 'release;latest'
-  - 'release;5.3.2'
```

Comment:
`Latest` should test the 5.x versions

Reply:
6.x is upcoming so we'll need to remember to add the last 5.x version here at that time.